content/docs/csidriver/installation/helm/unity.md

```diff
@@ -88,7 +88,7 @@ Install CSI Driver for Unity XT using this procedure.
 
 *Before you begin*
 
- * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.5.0 https://github.com/dell/csi-unity.git```, as a pre-requisite for running this procedure.
+ * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.6.0 https://github.com/dell/csi-unity.git```, as a pre-requisite for running this procedure.
```

Comment:
Move "As a prerequisite to running this procedure," to the front of this sentence.

Reply:
Done